spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/MessagingMessageListenerAdapter.java

```diff
@@ -638,12 +638,12 @@ private void sendReplyForMessageSource(Object result, String topic, Message<?> s
 			builder.setHeader(this.correlationHeaderName, correlationId);
 		}
 		setPartition(builder, source);
+		setKey(builder, source);
 		this.replyTemplate.send(builder.build());
 	}
 
 	protected void asyncSuccess(@Nullable Object result, String replyTopic, Message<?> source,
 			boolean returnTypeMessage) {
-
```

Comment:
This is really a good style to make code more readable.
There is might no reason to do that for single-like method header, but if it is multi-line, then better to separate body for easy review.
Please, revert this change and fix your IDE to preserver such a blank line on refactoring.